Source/ARTCrypto.h

```diff
@@ -44,11 +51,30 @@ NS_ASSUME_NONNULL_BEGIN
 
 @end
 
+/**
+ The ARTCrypto object ensures that message payloads are encrypted, can never be decrypted by Ably, and can only be decrypted by other clients that share the same secret symmetric key.
+ */
 @interface ARTCrypto : NSObject
 
-+ (ARTCipherParams *)getDefaultParams:(NSDictionary *)values;
+/**
+ Retrieves, or optionally sets, the `ARTCipherParams` for the channel.
```

Comment:
I don't think I agree with this description. My understanding is that `getDefaultParams` is a factory method for creating an `ARTCipherParams` instance. I don't think it queries or manipulates a channel in any way (bear in mind that this is a class method, so there isn't even a "the channel" to speak of here).

Reply:
Is this more accurate?

```
Creates an `ARTCipherParams` object to use for encryption. 
```